app/helpers/application_helper/button/iso_datastore_new.rb

```diff
@@ -9,6 +9,6 @@ def disabled?
   private
 
   def no_ems_without_iso_datastores?
-    !ManageIQ::Providers::Redhat::InfraManager.any_without_iso_datastores?
+    ExtManagementSystem.supporting(:create_iso_datastore).select { |ems| ems.supports?(:create_iso_datastore) }.none?
```

Comment:
curious why this does supporting, then another check for supports?

Reply:
`.supporting` only checks the class level supports (https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/supports_feature_mixin.rb#L134-L136)
We need to invoke the instance level as well (the block portion of supports :feature {})

Reply:
Had discussed with @kbrock and believe we came to the top-level method like this would just do class level checks for performance, and callers could choose to do instance level checks in addition if it was necessary